pkg/generic/descriptor/descriptor.go

```diff
@@ -47,11 +40,22 @@ type FieldDescriptor struct {
 	Type         *TypeDescriptor
 	HTTPMapping  HTTPMapping
 	ValueMapping ValueMapping
+	GoTagOpt     *GoTagOption
+}
+
+type GoTagOption struct {
```

Comment:
Pls add comment. And I am confused about why need two options?

Reply:
@AsterDY Got it. The reason why IsGoAliasDisabledSetByUser is necessary is that when this option is specified by user, the value of env var needs to be overwritten. Since IsGoAliasDisabled is of type bool, without IsGoAliasDisabledSetByUser, it would be impossible to distinguish whether the option was explicitly set by the user or not.

Reply:
This logic is just about your implementation, not about usage. It shouldn't be exposed to users, which only confuses them. What's more, why can't only one field override the env variable? I think it's applicable to writing codes instead of using two fields?

Reply:
LGTM